deanhaleem/mario-clone-ts
# Fonts(assets)[#8]: add emulogic font and display test text

### Github Issue

#8 

### Description

Add the emulogic font and displayed a random message on the screen.

**Note: you need to refresh in order to see it. Need to fix that later.**

### Testing Instructions

- `npm run dev`
- Notice the text on the screen
  - May need to refresh. Tech debt already smh
